### PR TITLE
Redirect section pages

### PR DESCRIPTION
**Pull request by aintgoin2goa**
Nows using the url mapper to redirect any url recognized as belonging to the old site

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,10 @@
 app := ft-next-redirector
 PORT := 3009
 ROUTER := $(shell which next-router)
+BIN := ./node_modules/.bin/
```
Comment (matthew-andrews) [inline Makefile]: this really shouldn't be necessary

Comment (matthew-andrews) [inline Makefile]: check your `$PATH` (2nd bullet in http://financial-times.github.io/next/docs/developer-guide/#common-errors)

File: Makefile [hunk 2]
```diff
@@ -22,14 +24,14 @@ run-router:
 	export redirector=${PORT}; export PORT=5050; export DEBUG=proxy; next-router;
 
 test:
-	@./node_modules/.bin/mocha ./test/
+	./node_modules/.bin/mocha ./test/
```
Comment (matthew-andrews) [inline Makefile]: could we add `next-build-tools verify`?

Comment (matthew-andrews) [inline Makefile]: [might need to upgrade to nbt 1.8+

**Comment by aintgoin2goa** [comment]
ok, is all fixed can I merge now?

**Pull request merged**